src/main/java/com/juniorstart/juniorstart/email/MailService.java

```diff
@@ -0,0 +1,57 @@
+package com.juniorstart.juniorstart.email;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.mail.MailException;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.MimeMessageHelper;
+import org.springframework.mail.javamail.MimeMessagePreparator;
+import org.springframework.stereotype.Service;
+
+/** Service class for sending emails 10-09-2020.
```

Comment:
to samo z datą

Reply:
zależy od koncepcji. jak robię kurs guru to on ma atuomat co mu od razu wpisuje date.